modules/account-lib/test/unit/coin/near/transaction.ts

```diff
@@ -53,7 +53,7 @@ describe('Near Transaction', () => {
     it('a signed transfer transaction', async () => {
       tx.fromRawTransaction(NearResources.rawTx.transfer.signed);
       const explain = tx.explainTransaction();
-      explain.id.should.equal('CEpsBC4fA64phQMfDnNzJqLFiDmgMhyAeu9vqN18EExH');
```

Comment:
why was near impacted?

Reply:
investigating and will revert this change when solved